modules/src/lib.rs

```diff
@@ -15,19 +15,31 @@
 )]
 #![forbid(unsafe_code)]
 
-//! Implementation of the following ICS modules:
+//! This library implements the _I_nter_B_lockchain _C_ommunication (IBC) protocol in Rust. IBC is
+//! a distributed protocol that enables communication between distinct sovereign blockchains.
+//! Loose analogies may be drawn between the IBC protocol and the TCP/UDP protocols that enable
+//! communication over the internet via packet streaming. Indeed, IBC also encodes the notion of
+//! ordered and unordered packet streams.
 //!
-//! - ICS 02: Client
-//! - ICS 03: Connection
-//! - ICS 04: Channel
-//! - ICS 05: Port
-//! - ICS 07: Tendermint Client
-//! - ICS 18: Basic relayer functions
-//! - ICS 23: Vector Commitment Scheme
-//! - ICS 24: Host Requirements
-//! - ICS 26: Routing
-//! - Applications:
-//!    - ICS 20: Fungible Token Transfer
+//! At its highest level, IBC can be categorized into on-chain modules and off-chain processes.
+//! On-chain modules, so called because they are the building blocks upon which IBC chains are
```

Comment:
```suggestion
//! The present crate `ibc` is a Rust implementation of the on-chain modules.
//! On-chain module, so called because it is the main building blocks upon which IBC chains are
```